Dockerfile

```diff
@@ -93,9 +93,20 @@ RUN cd /code/timemachine && git init . && pre-commit install-hooks
 COPY ci/requirements.txt /code/timemachine/ci/requirements.txt
 RUN pip install --no-cache-dir -r timemachine/ci/requirements.txt
 
-FROM tm_base_env AS timemachine
+# Dev container that contains the cuda developer tools
+FROM tm_base_env AS timemachine_dev
```

Comment:
Is the idea to leave `timemachine_dev` and define a new target in case the former is useful for debugging? (versus deleting the extra dev stuff not needed at runtime; seems fine to me, was just curious)

Reply:
Got it, I think that makes sense. I'd missed that the new target was using a different base image.